### ISSUE TITLE
Chargement long des annonces

### ISSUE THREAD

**Issue by c-geek** [c-geek/gannonce-ui#1]
Avec les 15 annonces déjà disponibles, le chargement se fait long.

Le fait que les images soient toutes chargées directement en JSON ralentit considérablement leur téléchargement et l'affichage de la liste d'annonce, car la liste ne s'affiche qu'une fois tout le JSON téléchargé.

Deux choses à faire : 

* ne plus envoyer les images dans le JSON de la requête `/announces`
* stocker un aperçu de la 1ère image lors de la soumission au serveur, puis restituer cet aperçu dans la requête JSON `/announces`. On peut aisément diviser la taille du téléchargement par 10.

**Comment by blavenie** [comment]
oui, pour Cesium+ et Gchange je ne télécharge pas l'image de miniature avec le reste du JSON, mais simplement une URL, qui est ensuite distribuée par le noeud ES (ton pod). On gagne beaucoup et en plus on profite du cache du navigateur ! (très utile pour les photo de profile, par exemple, qui se répètent souvent)

**Comment by c-geek** [comment]
Ceci dit, j'ai noté l'issue ici mais ne compte pas la réparer. Sauf rémunération ! 1000 Ğ1 pour environ 1 journée de travail estimé.
